Documentation/trying-out-rkt.md

```diff
@@ -161,74 +186,103 @@ Trusting "https://coreos.com/dist/pubkeys/aci-pubkeys.gpg" for prefix "coreos.co
 Added key for prefix "coreos.com/etcd" at "/etc/rkt/trustedkeys/prefix.d/coreos.com/etcd/8b86de38890ddb7291867b025210bd8888182190"
 ```
 
-For more information, see the [detailed, step-by-step guide for the signing procedure](signing-and-verification-guide.md).
+For more information, see the [detailed, step-by-step guide for the signing procedure][signguide].
+
+#### Fetching the ACI
 
-Now that we've trusted the CoreOS public key, we can fetch the ACI using `rkt fetch`:
+Now that the CoreOS public key is trusted, fetch the ACI using `rkt fetch`. This step doesn't need root privileges if the rkt host has been [configured for privilege seperation][privsep-heading]:
```

Comment:
Might be worth adding a quick reference to https://github.com/appc/spec/blob/master/spec/discovery.md to de-mystify some more of the "magic".